docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,401 @@ HTTP Response
       "status": 200
     }
 
+
+Creating a New User
+********************
+
+This endpoint exposes the Ledger API's :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>`.
```

Comment:
```suggestion
This endpoint exposes the Ledger API's :ref:`CreateUser RPC <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>`.
```

Reply:
Can you elaborate on that and whether it should be done for every reference?